<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vSimulation</class>
 <widget class="QWidget" name="vSimulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>6</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to your dream NBA vacation!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="foodGroupBox">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>46</y>
     <width>330</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="welcomeLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You are now at: </string>
    </property>
   </widget>
   <widget class="QLabel" name="currArenaLabel_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="souvenirTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>310</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="nextDestButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#cccccc;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Next Destination</string>
    </property>
   </widget>
   <widget class="QLabel" name="currDestCostLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>365</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current Souvenir Cost: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="currDestCostLine">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;* Select souvenirs for this destination below and click Confirm.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmSouvenirButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#cccccc;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>546</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#cccccc;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Abort Simulation</string>
   </property>
  </widget>
  <widget class="QLabel" name="goodbyeLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>506</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Thank you for traveling with Team Mudkip. Have a nice day!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>376</y>
     <width>220</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="totalDistLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TOTAL DISTANCE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalDistLine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="totalCostLabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>66</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TOTAL COST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalCostLine">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>85</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>kilometers</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>US dollars</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="endVacationButton">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>546</y>
     <width>130</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffa500;
border-style: solid;
border-color: black;
border-width: 2px;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Leave Simulation</string>
   </property>
  </widget>
  <widget class="QListWidget" name="destinationWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>46</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
